commit ce76851f4092e523578264e95c64eeef6adf83fe
Author: Taylor Hornby <havoc@defuse.ca>
Date:   Wed Sep 9 16:45:53 2015 -0600

    Add a hook for prepending to CXXFLAGS which will greatly simplify zerocashd depends packaging.

diff --git a/Makefile b/Makefile
index cbabc5c..7bb5f67 100644
--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,8 @@
 #* @copyright  MIT license (see LICENSE file)
 #*******************************************************************************/
 
-CXXFLAGS += -O2 -Wall -Wextra -Wno-unused-parameter -Wno-comment -march=native -mtune=native -std=c++11
+# CXXFLAGS_PREFIX allows the make caller to prepend CXXFLAGS.
+CXXFLAGS += $(CXXFLAGS_PREFIX) -O2 -Wall -Wextra -Wno-unused-parameter -Wno-comment -march=native -mtune=native -std=c++11
 
 DEPSRC=depsrc
 DEPINST=depinst
